core/src/test/java/org/bitcoinj/protocols/payments/PaymentSessionTest.java

```diff
@@ -216,8 +215,8 @@ public void testWrongNetwork() throws Throwable {
                 .build();
     }
 
-    private class MockPaymentSession extends PaymentSession {
```

Comment:
I ran this refactoring on the entire project via 7f38154244315cfcc869f4bd72daea9498567b3a.

Reply:
Thanks!